assets/data/categories_pl.json

```diff
@@ -4391,5 +4391,112 @@
       "Śmieje się jak głupi do sera",
       "Święta, Święta i... po Świętach"
     ]
+  },
+  {
+    "id": "MCU",
+    "image": "mcu.png",
+    "name": "Marvel Cinematic Universe",
+    "questions": [
+      "Avengers",
+      "Tony Stark",
+      "Iron Man",
+      "Steve Rogers", 
+      "Captain America",
+      "Bruce Banner",
+      "Hulk",
+      "Natasha Romanoff",
+      "Czarna wdowa (Black Widow)",
+      "Thor",
+      "Loki",
+      "Odin",
+      "Frigga",
+      "Hela",
+      "Mjölnir",
+      "Storm Breaker",
+      "Heimdall",
+      "Bifröst",
+      "Ragnarok",
+      "Valkyrie",
+      "Asgard",
+      "Sakaar",
+      "Dziewięć światów (Nine Realms)",
+      "Clint Barton",
+      "Hawkeye",
+      "Phil Coulson",
+      "Maria Hill",
+      "Nick Fury",
+      "Pepper Potts",
+      "Happy (Hogan)",
+      "James Rhodes",
+      "War Machine",
+      "Falcon",
+      "Sam Wilson",
+      "Wanda Maximoff",
+      "Scarlet Witch",
+      "Quicksilver",
+      "J.A.R.V.I.S.",
+      "Vision",
+      "Ultron",
+      "Stephen Strange",
+      "Peter Parker",
+      "Spider-Man",
+      "Bucky (James Barnes)",
+      "Winter Soldier",
+      "Wojna bohaterów (Civil war)",
+      "Sokovia",
+      "Strażnicy Galaktyki",
+      "Star Lord",
+      "Peter Quill",
+      "Rocket",
+      "Groot",
+      "Gamora",
+      "Drax the Destroyer",
+      "Nebula",
+      "Mantis",
+      "The Collector",
+      "Knowhere"
+      "Thanos",
+      "Pstryknięcie (The snap)",
+      "T'Challa",
+      "Black Panther",
+      "Okoye",
+      "Shuri",
+      "Wakanda",
+      "Dora Milaje",
+      "Vibranium",
+      "Antman",
+      "Osa",
+      "Scott Lang",
+      "Hope van Dyne",
+      "Hank Pym",
+      "Wymiar kwantowy (Quantum realm)",
+      "Chitauri",
+      "Kree",
+      "Skrull",
+      "Malekith",
+      "Rękawica",
+      "Kamienie nieskończoności",
+      "Kamień mocy",
+      "Kamień umysłu",
+      "Oko Agamotto",
+      "Kamień czasu",
+      "Aether",
+      "Tessaract",
+      "Kamień przestrzeni",
+      "Kamień rzeczywistości",
```

Comment:
You can add `Aether` here in brackets instead of placing it as separate word.

Reply:
Good point! They are combined now.